site/pages/api/nextdoor-callback.api.ts

```diff
@@ -0,0 +1,42 @@
+import { NextApiRequest, NextApiResponse } from "next";
```

Comment:
I don't think this is necessary as we did not do this for hootsuite / twitter if they click connect again the token will be overriden

Reply:
Some action is needed because with the way the tokens are set up if a user tries to set up two unique nextdoor accounts the first one that was added will stop working because of how we have set up the tokens. I know we say that an org should only have one nextdoor account but we can't let our code assume that, we should either be defensive in our code and block multiple nextdoor accounts or write our code in such a way that we can accommodate multiple accounts.

For hootsuite tokens we store the hootsuite user id against the token to give it a unique identifier. So what we could do for a simple fix is pass through the nextdoor user id to `getNextdoorSsmKey` to make sure it's unique per nextdoor account. I'll add a comment with a suggested code change.

Reply:
done